### PR TITLE
Made components border radius to use css variables.

### PR DESCRIPTION
**Pull request by m0ksem**
## Description
Unified borders:
```css
  --va-border-radius-sm: 0.25rem;
  --va-border-radius-md: 0.375rem;
  --va-border-radius-lg: 0.5rem;
  --va-border-radius-circle: 50%;
  --va-border-radius-circle-fixed: 100rem;
```

Removed such border-radiuses as 0.2rem, 0.3rem, 0.4rem, because they are not integral numbers.

Also use `--va-border-radius-circle-fixed: 100rem` which makes rounded corners for any size bottom (lower than 100rem which is impossible to have)

Please help me to review all these changes. Something can be broken.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-alert/_variables.scss [hunk 1]
```diff
@@ -1,42 +0,0 @@
-:root {
```
Comment (asvae) [inline packages/ui/src/components/va-alert/_variables.scss:1]: Deletion of this file doesn't seem intended to me.

Comment (asvae) [reply packages/ui/src/components/va-alert/_variables.scss:1]: 😿 

image

File: packages/ui/src/components/va-app-bar/VaAppBar.demo.vue [hunk 2]
```diff
@@ -144,7 +144,7 @@ export default {
 
 <style>
 .frame {
-  border-radius: 1rem;
+  border-radius: var(--va-border-radius);
```
Comment (asvae) [inline packages/ui/src/components/va-app-bar/VaAppBar.demo.vue:147]: Doesn't seem to be declared anywhere, so unlikely to work.
`--va-border-radius`

File: packages/ui/src/components/va-popup/popup/VaPopup.vue [hunk 3]
```diff
@@ -1,267 +0,0 @@
-<script>
```
Comment (asvae) [inline packages/ui/src/components/va-popup/popup/VaPopup.vue:1]: Doesn't appear to be part of this MR?

File: packages/ui/src/components/va-timeline/VaTimelineSeparator/_variables.scss [hunk 4]
```diff
@@ -1,19 +0,0 @@
-:root {
```
Comment (asvae) [inline packages/ui/src/components/va-timeline/VaTimelineSeparator/_variables.scss:1]: PRobably also not intended?

**Review by asvae — CHANGES_REQUESTED** [review]
Looks pretty dangerous, removes a bunch of random stuff.
Border related changes look fine to me though.

**Comment by m0ksem** [comment]
Looks like I need to update this PR with current HEAD. :(

**Comment by m0ksem** [comment]
This PR requires a lot of QA 😅

**Comment by m0ksem** [comment]
Moved to another task.

**Pull request closed without merging**